.vale/styles/Vocab/word_list/accept.txt

```diff
@@ -78,3 +78,27 @@ AMIs
 [Cc]ancelation
 Brotli
 zlib
+datetime
+GZip
+videogame
+toc
+gcp
+faq
+synon
+npm
+html
+Maneu
+Hoxie
+Bleve
+Yonik
+Seeley
+MBs
+url
+masterKey
+SDK's
```

Comment:
Is it necessary to specify the `s`? If we simply changed the earlier `SDKs` to `SDK`, would that solve the problem? Not sure how smart this rule is (i.e. does it form plurals of words added to the list and ignore those too?)

Mostly curious to better our understanding of Vale rules—not a problem if `SDK's` makes it onto the list.

Reply:
>does it form plurals of words added to the list and ignore those too?

It does not. So if `SDK` isn't a word, neither is `SDKs` or `SDK's`.

Reply:
I'm confused by your answer. It implies that if `SDK` is added to the list, `SDKs` and `SDK's` should also be accepted.

Reply:
Sorry if it was clear. If `SDK` isn't a word according to Vale, neither is `SDKs`, `SDK's`, or any other version of `SDK`. We will have to add all variations of the word separately to `accept.txt`.

Reply:
Huh. I think there is still a little miscommunication—I am wondering about what would happen if `SDK` _is_ a word according to Vale, not if it _isn't_. 

It would seem logical to me that if SDK is defined as a word, `SDK's` would be accepted as a natural grammatical extension of that word. Less sure about `SDKs` but it may also work. Could you test this, please?

Reply:
If it is a word, then we don't need to add `SDKs`, `SDK's` or any other variation